src/System.Net.Ping/tests/FunctionalTests/PingTest.cs

```diff
@@ -435,6 +684,16 @@ public static void Ping_SendAfterDispose_ThrowsSynchronously()
 
         private static readonly int s_pingcount = 4;
 
+        private static void SendBatchPing(Func<Ping, PingReply> sendPing, Action<PingReply> pingResultValidator)
+        {
+            // create several concurrent pings
+            Task[] pingTasks = new Task[s_pingcount];
```

Comment:
Is this method complete? You create `pingTasks` but never use it.

Reply:
It's a copy/paste error from the async method, pingTasks shouldn't be there.  Will remove